lib/ublksrv.c

```diff
@@ -43,6 +43,15 @@
  */
 #define UBLKSRV_IO_IDLE_SECS    20
 
+/*
+ * N.B. PR_SET_IO_FLUSHER was added with Linux 5.6+; we include a
+ * definition here in case this component is being built on a
+ * system whose headers are < 5.6
+ */
+#if !defined(PR_SET_IO_FLUSHER)
```

Comment:
I'd suggest to replace the above change into the change in PR#34:

#if defined(PR_SET_IO_FLUSHER)
	if (prctl(PR_SET_IO_FLUSHER, 0, 0, 0, 0) != 0)
		syslog(LOG_INFO, "ublk dev %d queue %d set_io_flusher failed",
			q->dev->ctrl_dev->dev_info.dev_id, q->q_id);
#endif